libs/ast_generic/AST_generic_to_v1.ml

```diff
@@ -967,9 +967,9 @@ and map_pattern = function
   | PatKeyVal (v1, v2) ->
       let v1 = map_pattern v1 and v2 = map_pattern v2 in
       `PatKeyVal (v1, v2)
-  | PatUnderscore v1 ->
+  | PatWildcard v1 ->
       let v1 = map_tok v1 in
-      `PatUnderscore v1
+      `PatWildcard v1
```

Comment:
you don't need to change those, you could have kept generating `PatUnderscore.
On fact ast_generic_v1.atd is supposed to be stable across changes in AST_generic.ml because it was meant
to be used for external plugins that could leverage the generic AST in other languages than OCaml.
Anyway, fine for now, but next time you don't need to complicate your life and you can keep ast_generic_v1.atd unchanged and just adapt the conversion in ast_generic_to_v1.ml

Reply:
Makes sense. Thanks for the comment!